isis/src/base/apps/photomet/main.cpp

```diff
@@ -1758,6 +1758,12 @@ void IsisMain() {
   // determine how photometric angles should be calculated
   angleSource = ui.GetString("ANGLESOURCE");
 
+  if ((normName == "TOPO" || normName == "MIXED") && angleSource == "DEM") {
+    QString message = "The " + normName + " Normalized model is not recommended for use with the " + angleSource + " Angle Source option";
+    PvlGroup warning("Warning");
+    warning.addKeyword(PvlKeyword("Warning",message),Pvl::Replace);
+    Application::Log(warning);
+  }
```

Comment:
Using Pvl::Replace will overwrite any other "Warning" keywords in a group. Yes the group was just created, so why add the Replace at all? If additional tests similar to this one are added they should add an additional "Warning" keyword to this group. With the "Replace" only one will show in the log.

Reply:
Suggest changing the group name to "Warnings"

Reply:
I changed the pvlgroup name to Warnings and removed the Pvl::Replace.